source/fundamentals/indexes.txt

```diff
@@ -270,8 +268,23 @@ omitted because text indexes do not contain sort order.
 .. driver-content-end
 .. common-content-begin
 
-For a full explanation of text search with MongoDB, refer to :manual:`Text Indexes </core/index-text>` in the MongoDB
-manual.
+.. note:: A collection can only contain one text index. 
+
+   If you want to create a text index for multiple text fields, you need
+   to create an compound index to include them. A compound text index
```

Comment:
It's not that clear whether the pronoun "them" refers to "text index" or "multiple fields". You don't lose anything from the sentence by omitting "to include them".

```suggestion
   to create an compound index. A compound text index
```